config/routes.rb

```diff
@@ -86,6 +86,7 @@
       get :completed_simulations_count
       get :experiment_stats
       get :experiment_moes
+      get :experiment_results
```

Comment:
Miałem raczej na myśli to, żeby zniknęły w tym kontekście prefiksy 'experiments_'. Aliasowanie nie jest tym co miałem na myśli.